addons/test_mail/tests/test_mail_activity.py

```diff
@@ -390,6 +390,116 @@ def test_activity_mixin(self):
             self.assertFalse(self.test_record.activity_state)
             self.assertFalse(act2.exists())
 
+    @mute_logger('odoo.addons.mail.models.mail_mail')
+    def test_activity_mixin_not_only_automated(self):
+        rec = self.test_record.with_user(self.user_employee)
+
+        now_utc = datetime.now(pytz.UTC)
+        now_user = now_utc.astimezone(pytz.timezone(self.env.user.tz or 'UTC'))
+        today_user = now_user.date()
+
+        # Schedule activities and create manual activities
+        act_type_todo = self.env.ref('test_mail.mail_act_test_todo')
```

Comment:
Do we need such setup ? Normally creating / scheduling activities is already covered, as well as unlinking, so it seems we could simplify the setup: create in batch some manual activities, then schedule one type, unlink that type with automated, check manual + automated have been removed. It seems we could then lessen test size by testing only relevant part, considering a lot of asserts here cover code that is already covered :) .